Project-II-Calculator/calculator/src/components/CalculatorComponent/MainButton.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import ActionButton from '../ButtonComponents/ActionButton'
+import NumberButton from '../ButtonComponents/NumberButton';
+const buttonStyle = {
+    color: 'black',
+    width: '25%',
+    height: '40px',
+    fontSize: '30px'
+}
+
+const actionStyle = {
+    color: 'white',
```

Comment:
Good job using JS to write your CSS! You definitrly want to check <a href="https://www.styled-components.com/">styled components </a> library. It's an enhanced version of what you are doing here!